test/travis_script.sh

```diff
@@ -160,4 +162,19 @@ if [ -f .coverage ]; then
     coverage report
 fi
 
+echo '>>> Test 6. Reading manta file'
+
+
+./scripts/manage.sh import --add_raw_data \
+                   --dataset "Dataset 1" \
+                   --version "Version 1" \
+                   --add_mates \
+                   --assembly_id "GRCh37p13" \
```

Comment:
This value is connected to `var_call_ref` which is no longer used. `--assembly_id` is used when loading reference sets instead.